assets/large_language_models/rag/components/src/flow_creation.py

```diff
@@ -135,10 +136,15 @@ def get_connection_name(s):
 
 def get_deployment_and_model_name(s):
     """get_deployment_and_model_name."""
-    deployment_and_model = s.split("/deployment/")[1]
-    deployment_name = deployment_and_model.split("/model/")[0]
-    model_name = deployment_and_model.split("/model/")[1]
-    return (deployment_name, model_name)
+    model_type = s.split("://")[0]
+    if model_type == "open_ai":
```

Comment:
Are we sure the input is always string, and it consists lower case letters only? "Open_AI://..." is not a possible case?